src/main/java/seedu/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package seedu.duke;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected String by;
```

Comment:
Variable naming can be better if added some time related info? Similar to Event class as mentioned by @LiuFangrui 
```suggestion
    protected String deadlineTime;
```